### PR TITLE
Notes about the 1-year product vision, how we should use it, etc.

### PR DESCRIPTION
**Pull request by sqs**

### CODE REVIEW

**Review by chrispine — APPROVED** [review]
LGTM!

**Review by jjeffwarner — COMMENTED** [review]
LGTM

**Review by unknwon — APPROVED** [review]
👍

File: content/departments/engineering/index.md [hunk 1]
```diff
@@ -10,7 +10,18 @@ The Engineering department at Sourcegraph consists of:
...
+Notes:
+
+- This vision is valid through 2023-09. That technically makes it a 15-month vision (not 12-month) from when we adopted it.
+- When we say devs will use it "10+ times daily", that's the high-level intent but not strict success criteria.
+  - We want to track and increase frequency of usage, and we want it to approach and/or exceed 10 times daily for as many devs as possible.
+  - If there are kinds or groups of devs who don't use Sourcegraph frequently (much less 10+ times daily), we should figure out why and see if we can make Sourcegraph more useful to them.
+  - For example, if a dev works at a company where only some of the code they work with is on Sourcegraph, then it's less likely they'll find Sourcegraph useful 10+ times daily. The most valuable first step is probably to figure out how to get all of their code on Sourcegraph.
+- What is the relationship between "10+ times daily" and "dev time saved"?
+  - We want users to use Sourcegraph 10+ times daily _because_ it saves them time.
+  - We don't want to artificially increase frequency of usage through annoying "growth hack" techniques that don't save devs time.
```
Comment (rrhyne) [inline content/departments/engineering/index.md:22]: Can you clarify what you define as annoying growth hacks?  Any of these on that list? 

* inviting users
* notifications
* @mentions
* emails
* integrations with messaging platforms

**Pull request merged**